example-script-runtime-service/README.md

```diff
@@ -0,0 +1,45 @@
+# example-script-runtime-service
+
+_Example Script Runtime Service_ based on the [Alfresco Script Runtime Service](https://github.com/Alfresco/alfresco-script-runtime-service/tree/develop/alfresco-script-app-runtime).
+
+*To build and publish*:
+
+```bash
+export DOCKER_IMAGE_TAG=<branch>
+./build.sh
+./push.sh
+```
+
+
+*How to test the built image*
+
+``` bash
+docker run -it -p 127.0.0.1:8080:8080 \
+  --env SCRIPTCONFIGURATION_SCRIPTSDEFINITIONSDIRECTORYPATH=file:/maven/scripts/ \
+  --env PROJECT_MANIFEST_FILE_PATH=file:/maven/project.json \
+  --env KEYCLOAK_AUTHSERVERURL=https://identity.***/auth \
+  --env SPRING_DATASOURCE_URL=jdbc:postgresql://***.us-east-1.rds.amazonaws.com:5432/external-db-app \
+  --env SPRING_DATASOURCE_USERNAME=*** \
+  --env SPRING_DATASOURCE_PASSWORD=*** \
+  --env SPRING_RABBITMQ_HOST=*** \
```

Comment:
i am not sure we have a queue for this runtime service, @mmartinadanx ?

Reply:
I can confirm as script runtime is a connector